apps/web/components/team/MemberChangeRoleModal.tsx

```diff
@@ -37,6 +38,23 @@ export default function MemberChangeRoleModal(props: {
       memberId: props.memberId,
       role,
     });
+
+    if (role === "OWNER") {
```

Comment:
We should avoid comparing to plain strings when fit
`if (role === MembershipRole.OWNER) {`